src/client/mtna/archive-template.jade

```diff
@@ -18,6 +18,10 @@ div.relative(role="main"
   md-content.md-default-theme( flex md-scroll-y elem-click="state.collapse()" )
     // main content
 
+    p.lead(ng-if="state._location._queryString == '' && state._location._startDate == '' && state._location._endDate == '' ") Welcome to the archives! <br /> <br /> To get started, click the blue bar to start a search, or click the "+" icon below to add a record. <br / <br /> <br /> <br /> <small>Hint! You can save your searches by bookmarking the page.</small>
```

Comment:
You can make this nicer by adding a . and newline at the end of hte attributes list

```
p.lead(   ).
  Plain Text
```